media-sound/qpwgraph/qpwgraph-0.2.2.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake
+
+DESCRIPTION="Qt GUI to control the JACK Audio Connection Kit and ALSA sequencer connections"
+HOMEPAGE="https://gitlab.freedesktop.org/rncbc/qpwgraph"
+SRC_URI="https://gitlab.freedesktop.org/rncbc/qpwgraph/-/archive/v${PV}/${PN}-v${PV}.tar.gz"
```

Comment:
Since you use the `${PN}-v${PV}` combination twice in the ebuild, you might wanna think about defining `MY_P="${PN}-v${PV}"` and using that instead to improve readability. That's just a suggestion, though.